core/src/costs.rs

```diff
@@ -502,6 +502,21 @@ impl SyscallCosts {
     }
 }
 
+/// Memory pages costs.
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct PagesCosts {
```

Comment:
also need to update `TYPE_LIST` (add `PagesCosts`, `IoCosts`) and regenerate `core/src/gas_metering/schedule.rs`
https://github.com/gear-tech/gear/blob/b9511183910d6b1ff6a018a5cadb72dc784b4fee/utils/weight-diff/src/main.rs#L305

Reply:
Done